main.py

```diff
@@ -657,4 +662,4 @@ def scheduled_import():
     sched.start()
 
     logging.basicConfig(level=logging.DEBUG, format='%(asctime)s %(message)s')
-    app.run(debug=True)
+    app.run(debug=True) if not args.open else app.run(host='0.0.0.0')
```

Comment:
I'd put this on separate lines, as a normal if block. The conditional expression is usually used when you want to return a value and you want to avoid multiple assignment statements.
Also, `debug=True` should probably also be given in the open mode.